.github/workflows/_ci.yaml

```diff
@@ -612,34 +619,205 @@ jobs:
+        token-name: ${{ env.TOKEN_NAME }}
+
+    - name: Download logs from S3
+      id: log-s3
+      run: |
+        mkdir -p /tmp/axlearn-output
+        aws s3 cp s3://jax-toolbox-eks-output/axlearn/${{ env.DATE_TEST_RAN }}/summary.txt /tmp/axlearn-output/
+
+        passed_tests=$(grep -c ": PASSED" /tmp/axlearn-output/summary.txt || true)
+        failed_tests=$(grep -c ": FAILED" /tmp/axlearn-output/summary.txt || true)
         total_tests=$((failed_tests + passed_tests))
-        echo "TOTAL_TESTS=${total_tests}" >> $GITHUB_OUTPUT
-        echo "ERRORS=${errors}" >> $GITHUB_OUTPUT
-        echo "PASSED_TESTS=${passed_tests}" >> $GITHUB_OUTPUT
-        echo "FAILED_TESTS=${failed_tests}" >> $GITHUB_OUTPUT
-      ARTIFACTS: |
-        test-gemma.log
-    secrets: inherit
 
-  test-maxtext:
-    needs: build-maxtext
-    if: inputs.ARCHITECTURE == 'amd64' # no arm64 gpu runners
-    uses: ./.github/workflows/_test_maxtext.yaml
+        echo "Passed tests: $passed_tests"
+        echo "Failed tests: $failed_tests"
+        echo "Total tests: $total_tests"
+
+        echo "PASSED_TESTS=$passed_tests" >> $GITHUB_OUTPUT
+        echo "FAILED_TESTS=$failed_tests" >> $GITHUB_OUTPUT
+        echo "TOTAL_TESTS=$total_tests" >> $GITHUB_OUTPUT
+    - name: Generate sitrep
+      id: sitrep
+      if: ${{ !cancelled() }}
+      shell: bash -x -e {0}
+      run: |
+        # bring in utility functions
+        source .github/workflows/scripts/to_json.sh
+
+        badge_label='Axlearn EKS Unit'
+
+        total_tests=${{ steps.log-s3.outputs.TOTAL_TESTS }} \
+        failed_tests=${{ steps.log-s3.outputs.FAILED_TESTS }} \
+        passed_tests=${{ steps.log-s3.outputs.PASSED_TESTS }} \
+        errors="0" \
+        summary="All tests: $total_tests. Passed: $passed_tests. Failed: $failed_tests." \
+        badge_message="Passed $passed_tests out of $total_tests." \
+        badge_color="brightgreen"
+        if [ "$failed_tests" -gt 0 ]; then
+          badge_color="red"
+        fi \
+
+        to_json \
+          summary \
+          errors total_tests passed_tests failed_tests \
+          badge_label badge_color badge_message \
+        > sitrep.json
+
+        schemaVersion=1 \
+        label="${badge_label}" \
+        message="Passed $passed_tests out of $total_tests." \
+        color=$badge_color \
+        to_json schemaVersion label message color \
+        > "badge-axlearn-test"
+
+    - name: Upload artifacts
+      if: ${{ !cancelled() }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: "artifact-axlearn-test"
+        path: |
+          sitrep.json
+          "badge-axlearn-test"
+          summary.txt
+
+  # test-axlearn-fuji-1B-slurm:
+  #   needs: build-axlearn
+  #   if: inputs.ARCHITECTURE == 'amd64' 
+  #   uses: ./.github/workflows/_test_fuji_1B.yaml
+  #   with:
+  #     AXLEARN_DOCKER_IMAGE: ${{ needs.build-axlearn.outputs.DOCKER_TAG_FINAL }}
+  #   secrets: inherit
+
+  test-axlearn-fuji-7B-slurm:
```

Comment:
I suggest dropping this. The raison d'être of the Slurm testing here was multi-node testing, which is no longer provided by the Slurm cluster and *is* provided by the EKS one. Single-node A100 testing can be added without the Slurm layer (following the JAX unit test model).